spec/decorators/odk/condition_group_decorator_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "spec_helper"
+
+module Odk
+  describe ConditionGroupDecorator, :odk, :reset_factory_sequences, database_cleaner: :truncate do
```

Comment:
Are these spec flags required given that you're mocking?

Reply:
I'm making a form in the spec still.